packages/core/src/fs/project-reader.ts

```diff
@@ -43,7 +43,7 @@ export class ProjectReader {
     { mutate, tempDirName, ignorePatterns, incremental, incrementalFile, force }: StrykerOptions
```

Comment:
```suggestion
    { mutate, tempDirName, ignorePatterns, incremental, incrementalFile, force, htmlReporter, jsonReporter }: StrykerOptions
```

You can pick off the `htmlReporter` and `jsonReporter` options from the `StrykerOptions` here.